src/utils/api.js

```diff
@@ -13,9 +13,9 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-import liskJS from 'lisk-js';
+import api from 'lisk-js';
```

Comment:
This is a confusing name change. What's the reasoning?

Reply:
Probably me being confused with renaming it to the `api` module